app/uk/gov/hmrc/partnershipidentificationfrontend/views/helpers/CheckYourAnswersListBuilder.scala

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.partnershipidentificationfrontend.views.helpers
+
+import play.api.i18n.Messages
+import play.api.mvc.Call
+import uk.gov.hmrc.govukfrontend.views.Aliases.{Actions, SummaryListRow, Value}
+import uk.gov.hmrc.govukfrontend.views.viewmodels.content.Text
+import uk.gov.hmrc.govukfrontend.views.viewmodels.summarylist.{ActionItem, Key}
+import uk.gov.hmrc.partnershipidentificationfrontend.controllers.routes
+import uk.gov.hmrc.partnershipidentificationfrontend.models.PartnershipInformation
+
+import javax.inject.Singleton
+
+@Singleton
+class CheckYourAnswersListBuilder {
```

Comment:
I think this needs the `@Inject` annotation

Reply:
...mmmm ...... not sure it needed it if there is nothing to inject ..... from the doc "..Injectable constructors are annotated with @Inject and accept zero or more dependencies as arguments"  .... but in our case the constructor is not ...let's say .... injectable .... it seems to work. Maybe nothing to worry. I could double check activating the debug of com.google.inject to see what is happening behind the scene ... what do you think???